.circleci/config.yml

```diff
@@ -3,7 +3,7 @@
   jobs:
     test:
       docker:
-        - image: continuumio/miniconda3:4.6.14
```

Comment:
This seems fine to me, as long as we have a fixed version and tests are passing.